graph/api/src/main/java/org/jboss/windup/graph/model/resource/FileModel.java

```diff
@@ -43,7 +43,7 @@
     public static final String TYPE = "FileResource";
 
     public static final String FILE_NAME = "fileName";
-    public static final String FILE_PATH = "filePath";
+    public static final String FILE_PATH = "fileModel:filePath";
```

Comment:
Although, you also need to update the index reference in GraphContextImpl if you are going to change this value.